litho-core/src/main/java/com/facebook/litho/InternalNode.java

```diff
@@ -460,61 +510,101 @@ InternalNode flexBasisAuto() {
 
   @Override
   public InternalNode flexBasisPercent(float percent) {
+    if (mIsBuilt) {
+      throw new IllegalStateException(ILLEGAL_MODIFY_OPERATION);
+    }
+
     mPrivateFlags |= PFLAG_FLEX_BASIS_IS_SET;
     mYogaNode.setFlexBasisPercent(percent);
     return this;
   }
 
   @Override
   public InternalNode flexBasisAttr(@AttrRes int resId, @DimenRes int defaultResId) {
+    if (mIsBuilt) {
+      throw new IllegalStateException(ILLEGAL_MODIFY_OPERATION);
+    }
+
```

Comment:
Since the first line of this method calls `flexBasisPx(int)` the check for object creation is redundant. There are a few more cases like this below.